mysql-test/main/rpl_mysqldump_slave.test

```diff
@@ -86,3 +86,52 @@ DROP TABLE t2;
 
 
 --source include/rpl_end.inc
+
+#
+# MDEV-32611 Added test for mysqldump --delete-master-logs option
+#
+
+connection master;
+use test;
+
+--source include/have_binlog_format_row.inc
```

Comment:
if that's the issue, can't I use `--source include/have_log_bin.inc`, as you told me in [zulip](https://mariadb.zulipchat.com/#narrow/stream/118760-new-members/topic/MDEV-32611/near/402743054).
or as i have included this `--source include/have_binlog_format_row.inc` at the very end, will this still create a problem for the above test?

Reply:
Good connection! `have_log_bin.inc` is actually already included in this file though :) Looking through the include chain of `--source include/master-slave.inc` (seen at `<src_dir>/mysql-test/include/master-slave.inc`), you'll see it calls `--source include/rpl_init.inc`, which in-turn calls `--source include/have_log_bin.inc`. 

To elaborate a little more on how this works, when you run `./mtr main.rpl_mysqldump_slave`, you'll see 3 separate tests be run, each re-running the entire test with a different binlog format. By including `--source include/have_binlog_format_<format>.inc`, the test will _only_ run under those hard-coded binlog formats (note you can combine them). So its presence would reduce the test space.

Reply:
1. 
>we'll have to modify the output of our --exec $MYSQL_DUMP command to mask the binary log position. See my comment on that line for the details.

I guess you were talking about this?
`--replace_regex /MASTER_LOG_POS=[0-9]+/MASTER_LOG_POS=BINLOG_START/`

2. removed `--source include/have_binlog_format_row.inc` so that test can run for all three log format. didn't additionally added `have_bin_log.inc` cause it already there through `master-slave.inc`.